src/services/measures/attitude/client/attitude.js

```diff
@@ -1,25 +1,85 @@
-tash.namespace('Gauges.Widgets');
+tash.namespace("Gauges.Widgets");
 
```

Comment:
Sorry... my code editor has a plug-in that automatically normalizes code. I'll try and remember to disable it going forward.